pages/linux/at.md

```diff
@@ -0,0 +1,15 @@
+# at
+
+> Executes commands at a specified time.
+
+- Open `at` prompt to execute commands at specific time. When you're done, press Ctrl+D to save the schedule:
+
+`at {{hh:mm:ss}}`
+
+- Execute commands and email result:
```

Comment:
thank you, I just cut "such as `sendmail`" to have a little bit shorter description.